pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControlUtils.java

```diff
@@ -100,11 +91,15 @@ public void validatePermission(Optional<String> tableNameOpt, AccessType accessT
    * @param httpHeaders HTTP headers containing requester identity required by access control object
```

Comment:
Update the javadoc. It's not a table level.

Reply:
done